python/hail/dataset.py

```diff
@@ -5749,12 +5749,12 @@ def trio_matrix(self, pedigree, complete_trios=False):
         each with an ``id`` and ``annotations`` field. The ``annotations`` fields have the
         same schema as the column annotation schema of the input dataset.
```

Comment:
Change to: The schema of each ``annotations`` field is the column annotation schema of the input dataset.

(as written you're referring to the schema of the column annotation schema)